samples/Automation/onboard-to-automation-dsc/azurepolicy.json

```diff
@@ -225,20 +273,25 @@
                   }
                 },
                 "variables": {
-                  "automationAccountName": "[last(split(parameters('automationAccount'),'/'))]"
+                  "automationAccountName": "[last(split(parameters('automationAccount'),'/'))]",
+                  "location": {
```

Comment:
@bmoore-msft this workaround does handle the issue I was fighting around template validation. What is your opinion of this?